docs/spec.yml

```diff
@@ -220,24 +219,69 @@ paths:
             type: string
           required: true
           description: Date at which the record was created. Sort key of the DynamoDB table.
-        - in: body
-          description: The desired status code of the new tech record and the reason for the unarchival
-          schema:
-            type: object
-            required: status, reasonForUnarchiving
-            properties:
-              status:
-                type: string
-              reasonForUnarchiving:
-                type: string
-
+      requestBody:
+        description: The desired status code of the new tech record and the reason for the unarchival
+        content:
+          application/json:
+            schema:
+              type: object
+              required: [status, reasonForUnarchiving]
+              properties:
+                status:
+                  type: string
+                reasonForUnarchiving:
+                  type: string
       responses:
         '200':
           description: Returns the unarchived tech record with the supplied status
           content:
             application/json:
               schema:
-                $ref: "https://raw.githubusercontent.com/dvsa/cvs-type-definitions/v2.0.21/json-schemas/v3/tech-record/get/psv/complete/index.json"
+                $ref: "https://raw.githubusercontent.com/dvsa/cvs-type-definitions/v6.1.0/json-schemas/v3/tech-record/get/psv/complete/index.json"
+        '400':
+          description: Bad request
+        '401':
+          description: Not authenticated
+        '403':
+          description: Unauthorised Access
+        '404':
+          description: Operation not supported
+        '422':
+          description: Unprocessable request
+        '500':
+          description: Validation Error
+  '/v3/technical-records{systemNumber}/{createdTimestamp}':
+    patch:
+      summary: 'Updates and archives record'
+      tags:
+        - Update technical record
+      parameters:
+        - in: path
+          name: systemNumber
+          schema:
+            type: string
+          required: true
+          description: This represents the systemNumber of the vehicle. Partition key of the DynamoDB table.
+        - in: path
+          name: createdTimestamp
+          schema:
+            type: string
+          required: true
+          description: Date at which the record was created. Sort key of the DynamoDB table.
+      requestBody:
+        required: true
+        description: The tech record to be updated
+        content:
+            application/json:
+              schema:
+                $ref: "https://raw.githubusercontent.com/dvsa/cvs-type-definitions/develop/json-schemas/v3/tech-record/put/psv/complete/index.json"
```

Comment:
Do you need a rebase? This is showing as your change / commit, even though it was merged into develop 14/03.

Also - there appears to be 3 references to /develop in the spec file, rather than /v6.1.0, (this line for example).

Do you happen to know why that is the case?

I'm getting the following error in the IDE when viewing the spec file locally...

`Something went wrong...`
`Invalid JSON pointer: https://raw.githubusercontent.com/dvsa/cvs-type-definitions/develop/json-schemas/v3/tech-record/put/psv/complete/index.json`

Reply:
Removed the references to develop from the spec file. Attempted a rebase however it would appear the section of code in question was actually deleted in develop due to conflicts with Open API v3.